stl/inc/format

```diff
@@ -46,6 +46,11 @@ concept _Parse_spec_callbacks = requires(_Ty _At, basic_string_view<_CharT> _Sv,
     { _At._On_width(_STD declval<int>()) } -> same_as<void>;
     { _At._On_dynamic_width(_STD declval<int>()) } -> same_as<void>;
     { _At._On_dynamic_width(_STD declval<_Auto_id_tag>()) } -> same_as<void>;
+    { _At._On_precision(_STD declval<int>()) } -> same_as<void>;
+    { _At._On_dynamic_precision(_STD declval<int>()) } -> same_as<void>;
+    { _At._On_dynamic_precision(_STD declval<_Auto_id_tag>()) } -> same_as<void>;
+
+    /* { _At._On_type(_STD declval<_CharT>()) } -> same_as<void>; */

```

Comment:
Style nitpick: We now try to use `//` comments everywhere unless the comment is embedded in a line.

Reply:
not addressing as this whole comment gets uncommented in the followup PR (as it adds the rest of the basic_format_spec parsing.